cmd/config/internal/commands/source_test.go

```diff
@@ -21,25 +21,37 @@ func TestSourceCommand(t *testing.T) {
 	}
 	defer os.RemoveAll(d)
 
-	err = ioutil.WriteFile(filepath.Join(d, "f1.yaml"), []byte(`
-kind: Deployment
-metadata:
-  labels:
-    app: nginx2
-  name: foo
-  annotations:
-    app: nginx2
-spec:
-  replicas: 1
+	err = ioutil.WriteFile(filepath.Join(d, "f1.json"), []byte(`
```

Comment:
Would you add an e2e test under `internal/commands/e2e` for source?

Reply:
Also update `sink_test.go` to make sure the results are written back as json

Reply:
@pwittrock Updated sink_test.go, but it is adding single quotes to json string. Please verify if it's fine.

E2e test for source ? How would it be different from source_test.go

Reply:
https://github.com/kubernetes-sigs/kustomize/pull/2552/commits/0e5e2648b345e847698791d3ae2fbf99679db73a#diff-8503fb052377a7f4a04632912645b0ef

Reply:
It gives and extra level of broader coverage.  A single test should be fine, with sink_test.go covering most of the cases.

Reply:
Sure. I will add the e2e tests after your PR is merged so that I can reuse the methods. We can create a separate story and add them for all other commands as well.